torch/functional.py

```diff
@@ -391,9 +391,11 @@ def _meshgrid(*tensors):
     return _VF.meshgrid(tensors)  # type: ignore
 
 
-def stft(input, n_fft, hop_length=None, win_length=None, window=None,
-         center=True, pad_mode='reflect', normalized=False, onesided=True):
-    # type: (Tensor, int, Optional[int], Optional[int], Optional[Tensor], bool, str, bool, bool) -> Tensor
+def stft(input: Tensor, n_fft: int, hop_length: Optional[int] = None,
+         win_length: Optional[int] = None, window: Optional[Tensor] = None,
+         center: bool = True, pad_mode: str = 'reflect', normalized: bool = False,
+         onesided: Optional[bool] = None,
+         return_complex: Optional[bool] = None) -> Tensor:
     r"""Short-time Fourier transform (STFT).
 
     Ignoring the optional batch dimension, this method computes the following
```

Comment:
It'd be nice to start with a brief sentence describing the function and how it's modeled after librosa's stft before diving into the math. I know you're copying the current preamble but this is a good opportunity to improve on it.

Reply:
Added a brief comment. Let me know if you want more.

Reply:
It's great, thank you!